.github/workflows/bundle-schema.yml

```
@@ -0,0 +1,26 @@
+name: Bundle & Push JSON schema
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - 'schemas/**'
+
+jobs:
+  bundle-push-schemas:
+    runs-on: ubuntu-latest
+    name: bundle-and-push-schemas
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Bundle & Push JSON schema for application schemas
+        uses: ./.github/actions/bundle-schema
+        with:
+          schema-path: ./schemas/applications/schema/application.schema.json
+          schema: application.schema.json
+      - name: Bundle & Push JSON schema for component-descriptions schemas
+        uses: ./.github/actions/bundle-schema
+        with:
+          schema-path: ./schemas/component-descriptions/schema/component.schema.json
+          schema: component.schema.json
```

Could you also update the `$id` of the controller description schema and include it here?

```suggestion
          schema: component.schema.json
      - name: Bundle & Push JSON schema for controller-descriptions schemas
        uses: ./.github/actions/bundle-schema
        with:
          schema-path: ./schemas/controller-descriptions/schema/controller.schema.json
          schema: controller.schema.json
```